languages/python/flipper/lf.py

```diff
@@ -96,5 +96,5 @@ def getModule(name):
 	return module(name, None, byref(_module))
 
 def invoke(module, function, ret, arguments):
-	libflipper.lf_invoke(module.ref, c_byte(function), c_byte(ret), lf_ll_from_list(arguments))
+	libflipper.lf_invoke(lf_get_current_device(), module.ref, c_byte(function), c_byte(ret), lf_ll_from_list(arguments))
```

Comment:
Did you actually create a python binding for `lf_get_current_device()` or did you do a repository-wide search and replace? I'm having trouble seeing what this is invoking and how it works.

Reply:
I have a commit brewing to "fix the language bindings". I have this one fixe .